### ISSUE TITLE
V8 version seems to be stuck to 8.8.278.15 with 0.63.4-patch.1

### ISSUE THREAD

**Issue by vtoupet** [Kudo/react-native-v8#62]
I've install v8:
```
"react-native-v8": "^0.63.4-patch.1"
"v8-android-jit-nointl": "^9.88.0"
``` 
When launching, the console log shows me `V8 version is 8.8.278.15`

@Kudo Do you know what could be wrong?

**Comment by Kudo** [comment]
@vtoupet The version `8.8.278.15` is the true V8 version. The major version + minor version corresponds to chrome version, e.g. this version is used by chrome 88.

For the versioning on npm, since it is for react-native-v8, the major version is preserved to prevent react-native-v8 use incorrect version of v8-android.
E.g. `react-native-v8@0.63.4-patch.0` has peerDependency of `v8-android^8.84.0` and it is not compatible of `v8-android@9.88.0`
For the minor version, it is also correspond to chrome version.

In summary, `v8-android-jit-nointl@9.88.0` uses `V8 8.8.278.15`, your result is right.

**Comment by Kudo** [comment]
I am going to close this issue. Please feel free to reopen in case there are any other questions.
